main.rb

```diff
@@ -1,11 +1,9 @@
-require_relative 'person'
-require_relative 'teacher'
-require_relative 'student'
-require_relative 'book'
-require_relative 'rental'
 require_relative 'classroom'
+require_relative 'handlers'
 
 class App
+  include Handlers
+
   def initialize
     @classroom = Classroom.new('Microverse 2.0')
     @people = []
```

Comment:
This function needs not be in the  `main.rb`, the purpose of this file is only to start the program.

- [ ] Kindly refactor